From: Andre Przywara <andre.przywara@amd.com>
Subject: [PATCH 03/12] KVM: SVM: Add clean-bit for IOPM_BASE and MSRPM_BASE
References: FATE#309760
Git-commit: f56838e4c38cbbc428ae276783f5dbb225e58817
Patch-mainline: v2.6.38

This patch adds the clean bit for the physical addresses of
the MSRPM and the IOPM. It does not need to be set in the
code because the only place where these values are changed
is the nested-svm vmrun and vmexit path. These functions
already mark the complete VMCB as dirty.

Backport of f56838e4c38cbbc428ae276783f5dbb225e58817
(done by: Joerg Roedel <joerg.roedel@amd.com>)

Signed-off-by: Andre Przywara <andre.przywara@amd.com>
Acked-by: Bruce Rogers <brogers@suse.com>
---
 arch/x86/kvm/svm.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

Index: linux-2.6.32-SLE11-SP2/arch/x86/kvm/svm.c
===================================================================
--- linux-2.6.32-SLE11-SP2.orig/arch/x86/kvm/svm.c
+++ linux-2.6.32-SLE11-SP2/arch/x86/kvm/svm.c
@@ -144,6 +144,7 @@ static int nested_svm_check_exception(st
 enum {
 	VMCB_INTERCEPTS, /* Intercept vectors, TSC offset,
 	                    pause filter count */
+	VMCB_PERM_MAP,   /* IOPM Base and MSRPM Base */
 	VMCB_DIRTY_MAX,
 };
 
